### DISCUSSION TITLE
Tune bm25 parameters

### DISCUSSION THREAD

**Discussion by zixiliuUSC** [castorini/pyserini#1111] [Q&A] [answered]
I notice that you have an automatic algorithm to set b and k in bm25 algo, I wanna know do you a api to set these parameters our own?

**Comment by lintool** [comment, marked as answer]
You can set the BM25 parameters programmatically as follows:
https://github.com/castorini/pyserini/blob/master/docs/usage-interactive-search.md

Is this what you were asking about?
